src/api/client.go

```diff
@@ -14,7 +14,10 @@ import (
 	"strings"
 	"time"
 
+	"github.com/sirupsen/logrus"
```

Comment:
there is already a package level logger defined [here](https://github.com/skycoin/skycoin/blob/cfffd0a51140934df2a38b670a18ef7356270f26/src/api/http.go#L28) so just use that